ripple/ui_react_base
# Setup react-loadable

What is Done:
- Added react-loadable
- Added common AsyncLoading component to handle all cases in loading process
- Added example for async load base on event, in this case route change
- Added example for pre-load async component

How to test:
- `npm run build` and you can see more than one `js` file(chunk) has created
-  `serve -s build`
- Load your app and open dev panel and go to network tab
- Click on `About` and you see a new file gets downloaded
- Click on `Contact` no download should happen because is pre-loaded
